content/integrate/redis-data-integration/data-pipelines/transform-examples/redis-add-field-example.md

```diff
@@ -0,0 +1,171 @@
+---
+Title: Add new fields to a key
+alwaysopen: false
+categories:
+- docs
+- integrate
+- rs
+- rdi
+description: null
+group: di
+linkTitle: Add new fields
+summary: Redis Data Integration keeps Redis in sync with the primary database in near
+  real time.
+type: integration
+weight: 40
+---
+
+By default, RDI adds fields to
+[hash]({{< relref "/develop/data-types/hashes" >}}) or
+[JSON]({{< relref "/develop/data-types/json" >}}) objects in the target
+database that match the columns of the source table.
+The examples below show how to add extra fields to the target data with the
+[`add_field`]({{< relref "/integrate/redis-data-integration/reference/data-transformation/add_field" >}}) transformation.
+
+## Add a single field
+
+The first example adds a single field to the data.
+The `source` section selects the `customer` table of the
+[`chinook`](https://github.com/Redislabs-Solution-Architects/rdi-quickstart-postgres)
+database (the optional `db` value here corresponds to the
+`sources.<source-name>.connection.database` value defined in
+[`config.yaml`]({{< relref "/integrate/redis-data-integration/data-pipelines/data-pipelines#the-configyaml-file" >}})).
+
+In the `transform` section, the `add_field` transformation adds an extra field called `fullname`
+to the object, which is created by concatenating the existing `firstname` and `lastname` fields using
+an [SQL expression](https://www.simplilearn.com/tutorials/sql-tutorial/concat-function-in-sql).
```

Comment:
```suggestion
a [SQL expression](https://www.simplilearn.com/tutorials/sql-tutorial/concat-function-in-sql).
```

Reply:
Not too sure about this one. I think it's officially "S-Q-L" rather than "sequel"?

Reply:
[In my best Richard Dawson impression...] And the style guide says: https://developers.google.com/style/word-list#sql

Reply:
Well, who am I to argue with Richard Dawson? ;-)